src/SonarScanner.MSBuild.Shim/SonarScanner.Wrapper.cs

```diff
@@ -49,10 +46,14 @@ public virtual bool Execute(AnalysisConfig config, IAnalysisPropertyProvider use
         return InternalExecute(config, userCmdLineArguments, propertiesFilePath);
     }
 
-    public virtual bool ExecuteJavaRunner(AnalysisConfig config, IAnalysisPropertyProvider userCmdLineArguments, string exeFileName, string propertiesFileName, IProcessRunner runner)
+    internal virtual bool ExecuteJavaRunner(AnalysisConfig config,
```

Comment:
Changed to `internal` thanks to `[assembly: InternalsVisibleTo("DynamicProxyGenAssembly2")]`